lib/elixir/pages/Typespecs.md

```diff
@@ -106,6 +112,14 @@ Built-in type           | Defined as
 
 Any module is also able to define its own type and the modules in Elixir are no exception. For example, a string is `String.t`, a range is `Range.t`, any enumerable can be `Enum.t` and so on.
 
+### Maps
+
+The key types in maps are allowed to overlap, and if they do, the leftmost pair takes precedence. A map value does not belong to this type if contains a key that is not in the maps allowed keys.
```

Comment:
"[...] does not belong to this type if **it** contains [...]", missing a small "it" :)